src/components/Integrations/Shopify/Shopify.js

```diff
@@ -90,16 +93,43 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     </>
   );
 
+  const getSubscribersAmountFromAPI = async (listId, apikey) => {
+    const resultAPI = await dopplerApiClient.getListData(listId, apikey);
+    if (resultAPI.success) {
+      return resultAPI.value.amountSubscribers;
+    } else {
+      return null;
+    }
+  };
+
+  const updateSubscriberCount = async (list) => {
```

Comment:
Personally, I would put this code directly in `getShopifyData`, but it is only a personal opinion, so hard opinion about it.

Reply:
Debatable... so I'll pass.